Bottom: 3b257643d41f65cf3e2d1573355a1bfdc1b1dfe5
Top:    2fbac692b91c079a462a0e0f3ec257c4ad33d080
Author: Pavel Roskin <proski@gnu.org>
Date:   2013-11-04 10:57:14 -0500

Refresh of make

---

diff --git a/farm-clean b/farm-clean
index 1ab0d27..18e7aab 100755
--- a/farm-clean
+++ b/farm-clean
@@ -7,5 +7,5 @@ for tree in $farm/*; do
 	test -d $tree/Documentation || continue
 	cd $tree
 	echo "Cleaning $tree"
-	make distclean
+	$make distclean
 done
diff --git a/farm-compile b/farm-compile
index 3a175c6..f506c6f 100755
--- a/farm-compile
+++ b/farm-compile
@@ -26,7 +26,7 @@ for bintree in $farm/*; do
 	test -n "$arch" || continue
 	log=Log-`basename $bintree`
 	rm -f $log $log-ok $log-error
-	make clean KERNELRELEASE=0 >/dev/null
+	$make clean KERNELRELEASE=0 >/dev/null
 	echo -n "Checking $bintree "
 	if $make HOSTCC="gcc -DPATH_MAX=4096" CC="gcc-4.4 $ccflags" \
 	     ARCH=$arch KBUILD=$bintree KERNELPATH=$bintree C=2 \
